lib/full-site-editing/block-templates.php

```diff
@@ -291,6 +291,7 @@ function _gutenberg_build_template_result_from_post( $post ) {
  *
  *     @type array  $slug__in List of slugs to include.
  *     @type int    $wp_id Post ID of customized template.
+ *     @type string $area A 'wp_template_part_area' taxonomy value to filter by (for wp_template_part template type only).
```

Comment:
Nitpick: Could we align the descriptions too? :smile: 
```suggestion
 *     @type array  $slug__in List of slugs to include.
 *     @type int    $wp_id    Post ID of customized template.
 *     @type string $area     A 'wp_template_part_area' taxonomy value to filter by (for wp_template_part template type only).
```

Reply:
good call 😁